checklist.md

```diff
@@ -0,0 +1,53 @@
+## F3 Activation Procedure (FRC-0099)
+
+This document outlines the procedure for activating F3 on the Filecoin network, adhering to FRC-0099. The process involves a multi-step approach, leveraging a smart contract to ensure transparency and community consensus.
+
+1.  **Parameter Selection and Bootstrap Epoch Determination**
+    *   Goal: Establish the "good parameter set" for F3 and the initial epoch for its activation.
+    *   Action: Analyze nv25 passive testing results. Consider network performance, stability, and community input.
+    *   Output: A finalized set of F3 parameters and a proposed bootstrap epoch.
+
+2.  **Manifest Definition and Storage**
+    *   Goal: Formally define the parameters and bootstrap epoch for distribution.
+    *   Action:
+        *   Create a manifest file (e.g., JSON) that clearly defines all F3 parameters and the selected bootstrap epoch.
+        *   Commit the manifest file into the `f3-activation-contract` repository.
+    *   Output: A verifiable manifest file stored in the repository.
+    *   **Manifest Link:** \[*Link to manifest file in repository*]
+
+3.  **Communication and Coordination**
+    *   Goal: Ensure all stakeholders are informed and aligned.
+    *   Action:
+        *   Communicate the proposed F3 parameters, bootstrap epoch, and rationale to Lotus, Forest, and Venus implementation teams.
+        *   Establish the expected encoding scheme of the manifest so that it is the same across implementations.
+        *   Share the manifest file and the address of the smart contract.
+    *   **Smart Contract Address:** \[*F3 Activation Smart Contract Address*]
+
+4.  **Smart Contract Interaction using SAFE Transaction Builder**
```

Comment:
Just curious why you're thinking to use the SAFE Transaction Builder UI vs. do this in the CLI so it can be reviewed now and just copy/pasted to execute.

Reply:
SAFE Transaction Builder is used to propose the transaction within SAFE wallet, with the inputs coming from CLI.

Reply:
Got it.  I was wondering about using something like https://github.com/safe-global/safe-cli .  I quickly tried this but ran into python environment / dependency issues and didn't want to sink much into it.  I think we are fine to defer, but I think it's an "area of improvement" if this contract-based rollout strategy is used in the future.